r_basics_introduction/r_basics_introduction.md

```diff
@@ -423,107 +434,136 @@ Below these images, we'll explain what each part of the R Markdown includes in a
+
+Make sure you **include the quotes** around tidyverse. When to use quotes is a tricky topic that we will talk about more later in this module, but for now, keep in mind that you will *always* need to put quotes around the name of a package whenever using `install.packages()`.
+
+</div>
 
-So these are the first two R commands we've covered - `install.packages` and `library`.
+After you've installed the package, in order to use the functions that it provides, you also need to **load** the package into the computer's memory. This is done with the command `library`. So to enable all the functions in the "tidyverse" package, you type `library("tidyverse")`. Packages remain loaded until you quit or restart the R program. Every time you start a new R session you have to load each package that you want to use again, and you need to put your `library` statements in the R documents that you write, preferably near the top of the file.  We'll use functions within the `tidyverse` package in this module when we import data, so it's useful to learn about packages now.
 
-One of the great things about R is that there are tens of thousands of packages, and virtually all of them are free to download and use, including all the packages we use in our modules.
+Congratulations, you've just learned your first two R commands - `install.packages` and `library`.
+
+One of the great things about R is that there are tens of thousands of packages that are free to download and use.  
+
+
+## Importing Data
+
+![`The data analysis pipeline, which begins with Import, then Tidy, then three actions that form a circle that repeat: Visualize, Model, and Transform.  The pipeline ends with "Communicate".`](media/pipeline_import.png)<!-- style = "max-width:500px;"-->
+
+<figcaption style = "font-size: 0.8em;"> Image taken from *R for Data Science* (https://r4ds.had.co.nz/introduction.html)</figcaption>
+
+
+The first step in the data analysis pipeline is to bring in, or **import**, the data to your environment, which for the purposes of this module is R.
+
+In this module, we'll work with a fabricated (completely made up) data set consisting of COVID-19 laboratory test results from a hypothetical infectious disease diagnostics lab. This data is stored as a **CSV file**.
+
+So what's a .csv file? CSV stands for **comma-separated values**.  When we say this aloud, we just say each letter: C - S - V.  
+
+A .csv file is a plain text file, which means you can open it in a text editor and look at it.  This is different than how you may have opened .csv files in the past, using Microsoft Excel or another spreadsheet program.  Let's take a look at what the raw data looks like when we open it in a text editor.
+
+![when viewed as plain text, csv files are just values separated by commas within each "row", with each row returned on a new row. ](media/csv.png)<!-- style = "max-width:500px;" -->
```

Comment:
```suggestion
![when viewed as plain text, csv files are just values separated by commas within each "row", and each row returned on a new line. ](media/csv.png)<!-- style = "max-width:500px;" -->
```

I know this was my own suggestion last time... but reading it with fresh eyes made me realize it did not make as much sense as I thought it did lol